heron_control/config/robot_localization.yaml

```diff
@@ -15,10 +15,10 @@ use_control: false
 # -------------------------------------
 imu0: imu/data
 imu0_config: [false, false, false,
-              false, false, true,
               false, false, false,
               false, false, true,
```

Comment:
Do you want to consider the Z velocity?

Reply:
Good question.  I just applied the changes Dave made in RPSW-562, which apparently resolved the driving issue.  It shouldn't actually affect anything since the IMU can't detect linear velocity anyway.  I'll revert this.